dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -327,3 +334,239 @@ export const SignInGateSelector = ({
 		</>
 	);
 };
+
+// -------------------------------
+// Auxia Integration Experiment //
+// -------------------------------
+
+export const SignInGateSelector = ({
+	contentType,
+	sectionId = '',
+	tags,
+	isPaidContent,
+	isPreview,
+	host = 'https://theguardian.com/',
+	pageId,
+	idUrl = 'https://profile.theguardian.com',
+	switches,
+}: Props) => {
+	const abTestAPI = useAB()?.api;
+	const userIsInAuxiaExperiment = !!abTestAPI?.isUserInVariant(
+		'auxiaSignInGate',
+		'auxia-signin-gate',
+	);
+
+	if (!userIsInAuxiaExperiment) {
+		return SignInGateSelectorDefault({
+			contentType,
+			sectionId,
+			tags,
+			isPaidContent,
+			isPreview,
+			host,
+			pageId,
+			idUrl,
+			switches,
+		});
+	} else {
+		return SignInGateSelectorAuxia({
+			host,
+			pageId,
+			idUrl,
+		});
+	}
+};
+
+/*
+	Date: January 2025
+	comment group: auxia-prototype-e55a86ef
+	Author: Pascal
+
+	We are currently starting to evaluate a new approach for the decision to display the sign gate or not: an
+	external API, that will be returning that decision. The company offering that API is called Auxia and
+	can be found here: https://www.auxia.io
+
+	At this aim I have move the existing legacy version of the SignInGateSelector to SignInGateSelectorDefault,
+	and I am introducing SignInGateSelectorAuxia, which is the new component that will be using the Auxia API.
+
+	This work, until further notice is the implementation and evaluation of a prototype.
+
+	All comments related to this prototype comes under the same comment group: auxia-prototype-e55a86ef
+	(follow it if one day you want to decommission the entire prototype).
+
+	PRs for ab test definition:
+		- https://github.com/guardian/frontend/pull/27743
+		- https://github.com/guardian/frontend/pull/27744
+		- https://github.com/guardian/dotcom-rendering/pull/13197
+*/
+
+type PropsAuxia = {
+	host?: string;
+	pageId: string;
+	idUrl?: string;
+};
+
+/*
+	comment group: auxia-prototype-e55a86ef
+	Signature for the ShowSignInGateAuxia component.
+*/
+interface ShowSignInGateAuxiaProps {
+	setShowGate: React.Dispatch<React.SetStateAction<boolean>>;
+	signInUrl: string;
+	registerUrl: string;
+	gateVariant: SignInGateComponent;
+	host: string;
+}
+
+interface SDCProxyData {
+	shouldShowSignInGate: boolean;
+}
+
+/*
+	comment group: auxia-prototype-e55a86ef
+	Auxia version of the dismissGate function.
+*/
+const dismissGateAuxia = (
+	setShowGate: React.Dispatch<React.SetStateAction<boolean>>,
+) => {
+	setShowGate(false);
+};
+
+const fetchAuxiaDisplayDataFromProxy = async (): Promise<SDCProxyData> => {
+	const url = 'https://contributions.guardianapis.com/auxia';
```

Comment:
not essential now, but we can avoid hardcoding this url with `getContributionsServiceUrl` -
https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/contributions.ts#L202

Reply:
Nice! Will put in a subsequent PR

Reply:
Done: https://github.com/guardian/dotcom-rendering/pull/13230